uber/api.py

```diff
@@ -0,0 +1,30 @@
+from uber.common import *
+
```

Comment:
This is the format expected by the ``to_dict()`` method which Sideboard provides on anything which uses its ``@declarative_base`` decorator.  This is extensively documented at
https://github.com/magfest/sideboard/blob/master/sideboard/lib/sa/_crud.py#L1

The tl'dr version is that we generate a dictionary from the model's fields and can optionally tell it which fields to use, including which foreign key relationships it should follow.